engine-tests/src/test/java/org/terasology/documentation/apiScraper/util/ApiMethod.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.documentation.apiScraper.util;
+
+
+import java.util.Objects;
+
+/**
+ * Saves informations about methods and constructors to be used at the ApiComparator class
+ */
+public class ApiMethod {
+
+
+    private String className;
+    private String name;
+    private String returnType;
+    private String exceptionType;
+    private String parametersType;
+
+    /**
+     *
```

Comment:
Another spot where you could use a main javadoc description (or else any actual generated javadoc would end up with some empty boxes here and there)